static/sass/initial/_type.scss

```diff
@@ -57,30 +57,43 @@ h3 {
 a {
   #{$link} {
     @include text-underline('action');
-    color: inherit;
+    color: color('action');
     font-weight: bold;
     text-decoration: none;
     transition: color 150ms;
+
+    h1 &,
+    h2 &,
+    h3 & {
+      background-image: none; // remove link underline for headlines
+    }
   }
 
   #{$focus} {
-    color: color('action');
+    background-image: linear-gradient(color('action-hover'), color('action-hover'));
+    color: color('action-hover');
   }
 }
 
+/// @todo totally yucky, need a better class to hook into
+///   [data-module='rst-body'] is only for blog posts
+///   and no global inner-level class exists
```

Comment:
What's the goal here? I don't understand this comment. You just want a class to use instead of `data-region="main"`?